docs/README.md

```diff
@@ -20,11 +20,12 @@ where P is either the Dirichlet process or the Pitman-Yor process.
 
 - Serialization of the MCMC chains is possible using [Google's protocol buffers](https://developers.google.com/protocol-buffers).
 
-
-
 ## Installation
+
```

Comment:
This file is identical to the main README.md, is there a reason?

Reply:
I vaguely remember *moving* it to the `docs` subfolder because it wouldn't show up somewhere, but not *copying* it. Do we need both in the documentation? If not, we can delete one for sure